docs/Configuring-Orion/database/postgres_storage.sql

```diff
@@ -0,0 +1,7 @@
+/* Creates the database table needed when using PostgreSQL for Orion storage. */
+
+CREATE TABLE store (
+  key bytea,
+  value bytea,
+  primary key(key)
+);
```

Comment:
Where did this change come from?  It's not one I remember making.

Reply:
@joshuafernandes made it